docs/snippets/vue/portable-stories-jest-with-play-function.ts.mdx

```diff
@@ -0,0 +1,18 @@
+```ts
+// Button.test.ts
+import { test } from '@jest/globals';
+import { render } from '@testing-library/vue';
+import { composeStory } from '@storybook/vue3';
+
+import meta, { Primary } from './Button.stories';
+
+test('renders and executes the play function', async () => {
+  const PrimaryStory = composeStory(Primary, meta);
+
+  // First, render the story
+  render(PrimaryStory());
```

Comment:
If https://github.com/storybookjs/storybook/pull/26198 lands as is, the API has changed slightly

```suggestion
  render(PrimaryStory);
```